### PR TITLE
[skip ci] minor documentation changes

### PR DESCRIPTION
**Pull request by goddammit**
Please merge. More to come next week.

### CODE REVIEW

File: doc/Advanced-configuration.md [hunk 1]
```diff
@@ -1,24 +1,32 @@
-For advanced configuration use the following files: `ejabberd.cfg`, `vm.args` and `app.config`. The first one contains all pure XMPP settings, the second affects Erlang VM behaviour (performance tuning, node name) and third usually has to be modified to change low-level logging parameters. Since you've gotten this far, we assume you're already familiar with Erlang syntax.
+For advanced configuration use the following files:
+
+* `ejabberd.cfg` for pure XMPP settings,
```
Comment (michalwski) [inline doc/Advanced-configuration.md]: `for pure MongooseIM settings`. I know this doesn't sound well. On the other hand saying this is only about XMPP settings is not fully true.

File: doc/Advanced-configuration.md [hunk 2]
```diff
@@ -1,24 +1,32 @@
-For advanced configuration use the following files: `ejabberd.cfg`, `vm.args` and `app.config`. The first one contains all pure XMPP settings, the second affects Erlang VM behaviour (performance tuning, node name) and third usually has to be modified to change low-level logging parameters. Since you've gotten this far, we assume you're already familiar with Erlang syntax.
+For advanced configuration use the following files:
+
+* `ejabberd.cfg` for pure XMPP settings, 
+
+* `vm.args` to affect the Erlang VM behaviour (performance tuning, node name), 
+
+* `app.config` to change low-level logging parameters.
```
Comment (michalwski) [inline doc/Advanced-configuration.md]: and other Erlang applications as well.

File: doc/Advanced-configuration.md [hunk 3]
```diff
@@ -33,8 +41,8 @@ The tuple order is important, unless the no `host_config` option is set. Retaini
 * **alarms** (global)
```
Comment (michalwski) [inline doc/Advanced-configuration.md]: This is no longer true, the whole section about alarms can be removed.

File: doc/open-extensions/muc_light.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+Classic Multi-User chat, as described in XEP-0045, adds an IRC-like functionality to XMPP. It distinguishes between the affiliation list and the occupant list, where the latter is based on presences routed to the room from the client resource. While perfectly sufficient for desktop applications and relatively stable network connection, it does not exactly meet the challenges the mobile world it is facing. Modern mobile applications do not rely on presence information, as it can frequently change. The expected user experience not only differs from the IRC model, but also uses only a small subset of XEP-0045 features. The service described in this specification attempts to provide a complete solution for all common use cases of mobile groupchats.
```
Comment (michalwski) [inline doc/open-extensions/muc_light.md]: Could we adopt a rule "one sentence per line"? Formatting on github and readthedocs will remain the same, but editing will be so much easier, and diffs so much clearer.

File: doc/Advanced-configuration.md (hunk 1 above)
Comment (goddammit) [inline doc/Advanced-configuration.md]: hmm... that was in the original note. I'll think about rephrasing it, any ideas of what meaning I should convey?

File: doc/Advanced-configuration.md (hunk 2 above)
Comment (goddammit) [inline doc/Advanced-configuration.md]: the vm.args? Again, I only changed the layout, let me know what ideas I should convey in the reviewed version

File: doc/Advanced-configuration.md (hunk 3 above)
Comment (goddammit) [inline doc/Advanced-configuration.md]: cool:)

File: doc/open-extensions/muc_light.md (hunk 4 above)
Comment (goddammit) [inline doc/open-extensions/muc_light.md]: ack

**Comment by Nyco** [comment]
Awesome work!

**Comment by michalwski** [comment]
Thanks!

**Pull request merged**